app/app/tests/utils/utils.py

```diff
@@ -11,6 +11,14 @@ def random_lower_string() -> str:
     return "".join(random.choices(string.ascii_lowercase, k=32))
 
 
+def random_password() -> str:
+    return "".join(
+        random.choices(string.ascii_lowercase, k=3)
+        + random.choices(string.ascii_uppercase, k=3)
+        + random.choices(string.digits, k=3),
+    )
+
+
```

Comment:
hmm we will also need this wen start tests

Reply:
Start abhi se, you'll regret later.